profiling/nvprof-connector/kp_nvprof_connector.cpp

```diff
@@ -50,6 +50,11 @@
 #include <iostream>
 
 #include "nvToolsExt.h"
+#include "impl/Kokkos_Profiling_C_Interface.h"
```

Comment:
Should we define `Kokkos_Tools_ToolSettings` instead?

Reply:
That seems very likely to lead to ODR violations from mis-matched kokkos & kokkos-tools versions at some point no?

Reply:
I would be surprised if it did actually give an issue unless Kokkos prepend another boolean option.